### ISSUE TITLE
USER ISSUE: kick from server when destroying tree 

### ISSUE THREAD

**Issue by DubsTheFox** [StrangeLoopGames/EcoIssues#6915]
**Version:** 0.7.3.1 beta

We get kicked from the server when we try to destroy a tree that planted itself on our farm.

The following error message popped up:
```
Caught exception

<size=60,00%>Object reference not set to an instance of an object.

   at Eco.Simulation.Agents.Plant.Destroy()
   at Eco.Simulation.Agents.Tree.Destroy()
   at TreeEntity.Destroy()
   at TreeEntity.TryApplyDamage(INetObject damager, Single amount, InteractionContext context)
   at Eco.Gameplay.Items.WeaponItem.OnActLeft(InteractionContext context)
   at Eco.Mods.TechTree.AxeItem.OnActLeft(InteractionContext context)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Players.Player.PlayerInteract(InteractionInfo info)
</size>
```

**Comment by Ekkelus** [comment]
I got the same error. Two trees in the same spot somehow (in my case) and they appeared under a stockpile. Can't be removed.

**Comment by Dreepa** [comment]
https://github.com/StrangeLoopGames/EcoIssues/issues/3752

**Comment by Ekkelus** [comment]
This is not when chopping logs though, this is when trying to destroy a tree (in my case dead tree).

**Comment by MarjieAVolk** [comment]
We believe this is fixed in the next hotfix (#5619)
